### DISCUSSION TITLE
Can DeepMD read the output file of ORCA？

### DISCUSSION THREAD

**Discussion by berlye** [deepmodeling/dpdata#428] [General]
I used ORCA software to do calculations before, and I want to use it in combination with DeepMD in the future. Can these two kinds of software connect?

**Comment by njzjz** [comment]
You can connect them indirectly: ASE supports OCRA and dpdata supports loading from ASE atoms.

**Reply by berlye** [reply]
Thank you very much！

**Reply by berlye** [reply]
Hello, I would like to ask you another question. I used ORCA software to do QMMM calculations and got the corresponding atomic type and coordinates of the system（xyz）. Should I convert this data to ASE traj format or another format? Or what format do I want to convert the ORCAR software output log file to ASE?

**Reply by Yancy-Luke** [reply]
Hello，have you solved your problem？I have the same problem as you
